games-fps/urbanterror/urbanterror-4.3.2_p20180214.ebuild

```diff
@@ -0,0 +1,163 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+inherit eapi7-ver flag-o-matic toolchain-funcs xdg-utils
+
+DESCRIPTION="Hollywood tactical shooter based on the ioquake3 engine"
+HOMEPAGE="http://urbanterror.info https://github.com/mickael9/ioq3"
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/mickael9/ioq3.git"
+	EGIT_BRANCH="urt"
+else
+	COMMIT_ID="1a6c221f932e75eb278fd063884cd69610c5679d"
+	SRC_URI="https://github.com/mickael9/ioq3/archive/${COMMIT_ID}.tar.gz -> ${P}.tar.gz"
+	S="${WORKDIR}/ioq3-${COMMIT_ID}"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="+altgamma +client +curl debug mumble openal +opus server +skeetshootmod voip vorbis"
+REQUIRED_USE=" || ( client server )"
+
+PATCHES=( "${FILESDIR}"/${PN}-4.3-fix-build_system.patch )
+RDEPEND="
+	client? (
+		media-libs/libsdl2:=[X,sound,joystick,opengl,video]
+		mumble? ( media-sound/mumble:= )
+		openal? ( media-libs/openal:= )
+		opus? ( media-libs/opusfile:= )
+		vorbis? (
+			media-libs/libogg:=
+			media-libs/libvorbis:=
+		)
+	)
+	curl? ( net-misc/curl )
+	~games-fps/urbanterror-data-4.3.2
+	sys-libs/zlib:=[minizip]
+	virtual/jpeg:0
+"
+
+DEPEND="${RDEPEND}"
+
+pkg_pretend() {
+	if use client; then
+		if ! use openal && ! use opus && ! use vorbis; then
+			ewarn
+			ewarn "No sound implementation selected. Enable 'openal', 'opus' or 'vorbis' USE flag to get sound!"
+		fi
+	fi
+}
+
+src_configure() {
+	default
+
+	tc-export CC
+}
+
+src_compile() {
+	# Workaround for used zlib macro, which got renamed in Gentoo
+	# wrt bug #449510
+	append-cppflags "-DOF=_Z_OF"
+
+	local my_arch=$(usex amd64 "x86_64" "i386")
+
+	emake \
+		ARCH=${my_arch} \
+		DEFAULT_BASEDIR="/usr/share/urbanterror" \
+		BUILD_CLIENT=$(usex "client" 1 0) \
+		BUILD_SERVER=$(usex "server" 1 0) \
+		BUILD_BASEGAME=1 \
+		BUILD_MISSIONPACK=0 \
+		BUILD_GAME_SO=0 \
+		BUILD_GAME_QVM=0 \
+		BUILD_STANDALONE=1 \
+		SERVERBIN="Quake3-UrT-Ded" \
+		CLIENTBIN="Quake3-UrT" \
+		USE_RENDERER_DLOPEN=0 \
+		USE_YACC=0 \
+		BASEGAME="q3ut4"\
+		BASEGAME_CFLAGS="${CFLAGS}" \
+		USE_OPENAL=$(usex "openal" 1 0) \
+		USE_OPENAL_DLOPEN=$(usex "openal" 1 0) \
+		USE_CURL=$(usex "curl" 1 0) \
+		USE_CURL_DLOPEN=$(usex "curl" 1 0) \
+		USE_CODEC_VORBIS=$(usex "vorbis" 1 0) \
+		USE_CODEC_OPUS=$(usex "opus" 1 0) \
+		USE_MUMBLE=$(usex "mumble" 1 0) \
+		USE_SKEETMOD=$(usex "skeetshootmod" 1 0) \
+		USE_VOIP=$(usex "mumble" 1 0) \
+		USE_INTERNAL_LIBS=0 \
+		USE_LOCAL_HEADERS=0 \
+		USE_ALTGAMMA=$(usex "altgamma" 1 0)
+}
+
+src_install() {
+	local my_arch=$(usex amd64 "x86_64" "i386")
+	# docs from ioq3, not from UrbanTerror ZIP file
+	dodoc ChangeLog README.md README.ioq3.md md4-readme.txt
```

Comment:
Please add those files to the `DOCS` array outside `src_install` and call `einstalldocs` instead.